ocaml-lsp-server/test/e2e/__tests__/textDocument-codeAction.test.ts

```diff
@@ -143,4 +167,169 @@ let f (x : t) = x
       },
     ]);
   });
+
+  it("can annotate a function argument", async () => {
+    await openDocument(
+      outdent`
+type t = Foo of int | Bar of bool
+
+let f x = Foo x
+`,
+      "file:///test.ml",
+    );
+    let start = Types.Position.create(2, 6);
+    let end = Types.Position.create(2, 7);
+    let actions = await codeAction("file:///test.ml", start, end);
+    expect(actions).toMatchObject([
```

Comment:
Can you filter this list to only show the code action you're interested in?

Reply:
Solved. The diff displayed is less actionable (everything is on a single line), but it's not dramatic